shanoir-ng-datasets/src/main/java/org/shanoir/ng/shared/model/Center.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Shanoir NG - Import, manage and share neuroimaging data
+ * Copyright (C) 2009-2019 Inria - https://www.inria.fr/
+ * Contact us on https://project.inria.fr/shanoir/
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
+ */
+package org.shanoir.ng.shared.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * @author yyao
+ *
+ */
+@Entity
+@Table(name = "center")
```

Comment:
So we have 2 tables center, both in datasets and studies ?

Reply:
yes, like study and subject tables in ms datasets. they are duplicated via rabbitmq when idName of they are changed in the ms studies to keep the index up-to-date..